RecoHI/HiJetAlgos/plugins/UETableProducer.cc

```diff
@@ -0,0 +1,232 @@
+// system include files
```

Comment:
Files like this should normally be in a Cond* sybsystem package.
There is nothing reco related here.
Having incorrect placement also means the code review will not be appropriate.

@diguida @apfeiffer1 
plain DB writers show up occasionally in a few reco packages, I can sign off, but my knowledge of what's done in these files is limited.